pkg/deploy-manager/config.go

```diff
@@ -77,8 +104,17 @@ func (t *DeployManager) GetNamespace() string {
 	return InstallNamespace
 }
 
-// NewDeployManager is the way to create a DeployManager struct
+// NewDeployManager creates a DeployManager struct with default configuration
 func NewDeployManager() (*DeployManager, error) {
+	return newDeployManager(GetDefaultStorageClusterOptions())
+}
+
+// NewDeployManagerWithOptions creates a DeployManager struct with user provided configuration
+func NewDeployManagerWithOptions(opts *StorageClusterOptions) (*DeployManager, error) {
+	return newDeployManager(opts)
```

Comment:
This seems to just be a passthrough to a function with identical parameters... why? Should this just be the function currently being called?

Reply:
Obsolete with the update.